doc/edam/api.rst

```diff
@@ -31,6 +31,8 @@ Different EDA tools handle different subsets of files and are expected to ignore
 - vhdlSource-2008 : VHDL 2008 source
 - xci : Xilinx Vivado IP file
 - xdc : Xilinx Vivado constraint file
+- bba : Nextpnr-Xilinx chip data file
+- synthJson : Yosys-generated synthesys output file
```

Comment:
Typo s/synthesys/synthesis
Is bba specific to Nextpnr-Xilinx, or are they used for other chip types supported by Nextpnr?